packages/dds/sequence/src/defaultMap.ts

```diff
@@ -286,7 +281,7 @@ export class DefaultMap<T> {
 	}
 
 	public populate(json: string): void {
-		this.populateFromSerializable(JSON.parse(json) as IMapDataObjectSerializable);
+		this.populateFromSerializable(this.serializer.parse(json) as IMapDataObjectSerializable);
```

Comment:
This is surprising, based on the name `populateFromSerializable`. The object passed in there will have handles in non-serializable form